spec/api-app-spec.js

```diff
@@ -358,6 +358,28 @@ describe('app module', () => {
       })
       w = new BrowserWindow({ show: false })
     })
+
+    it('should emit remote-require event when remote.require() is invoked', (done) => {
+      app.once('remote-require', (event, webContents, moduleName) => {
```

Comment:
Can you please use `emittedOnce` here?

```js
const eventReceived = emittedOnce(app, 'remote-require')
<...>
const [, webContents, moduleName] = await eventReceived
expect(<...>)
```

Reply:
it won't work as I use `executeJavaScript` later

Reply:
@miniak You can delay the promise as described in @alexeykuzmin's sample.  E.g.

```js
const eventReceived = emittedOnce(app, 'remote-require')
await executeJavaScript(thing);
const [, webContents, moduleName] = await eventReceived
expect(<...>)
```